### PR TITLE
include cookies in subdomains

### PR DESCRIPTION
**Pull request by teleaziz**

### CODE REVIEW

File: packages/core/src/builder.class.ts [hunk 1]
```diff
@@ -91,12 +91,14 @@ function setCookie(name: string, value: string, expires?: Date) {
     }
 
     const secure = isBrowser ? location.protocol === 'https:' : true;
+    const nakedDomain = location.hostname.split('.').slice(-2).join('.');
```
Comment (steve8708) [inline packages/core/src/builder.class.ts:94]: this will break for `.co.uk` etc I think?

Comment (teleaziz) [reply packages/core/src/builder.class.ts:94]: true, I'll go the other way about it, omitting only the first sub-domain

Comment (mrkoreye) [reply packages/core/src/builder.class.ts:94]: Do we also need to modify the `this.setCookie` method to include domain for when the cookie is set via the `Cookies` class on the server? https://github.com/BuilderIO/builder/blob/master/packages/core/src/builder.class.ts#L2021

Comment (teleaziz) [reply packages/core/src/builder.class.ts:94]: yes @mrkoreye , already modified here https://github.com/BuilderIO/builder/pull/237/files#diff-5ed22976e6209bba5c2b4da86850f86dR59

Comment (mrkoreye) [reply packages/core/src/builder.class.ts:94]: oh lol how did i miss that 😆

**Pull request merged**